packages/console/src/ds-components/Table/index.tsx

```diff
@@ -164,7 +164,8 @@ function Table<
         </OverlayScrollbar>
       </div>
       <div className={styles.footer}>
-        {footer}
+        {/* Fall back to a div if footer is not provided to avoid layout shift. */}
+        {footer ?? <div />}
```

Comment:
Why fall back to div?

Reply:
dude the comment is right above

Reply:
I saw this one, but really wanna know the root cause, as this seems to be a hack

Reply:
Ah I see